pkg/executor/executortype/container/containermgr.go

```diff
@@ -240,7 +240,7 @@ func (caaf *Container) RefreshFuncPods(ctx context.Context, logger *zap.Logger,
 
 	funcLabels := caaf.getDeployLabels(f.ObjectMeta)
 
-	dep, err := caaf.kubernetesClient.AppsV1().Deployments(metav1.NamespaceAll).List(ctx, metav1.ListOptions{
+	dep, err := caaf.kubernetesClient.AppsV1().Deployments(f.Namespace).List(ctx, metav1.ListOptions{
```

Comment:
If the function is created in the default namespace and if the namespace is in the fission-function namespace, this may not work. Please revert this change for now.

Reply:
done